config/samples/backends/lvm-iscsi/lvm.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+    service: cinder
+  name: 99-master-cinder-lvm-losetup
+spec:
+  config:
+    ignition:
+      version:  3.2.0
+    Systemd:
+      Units:
+      - name: cinder-lvm-losetup.service
+        enabled: true
+        Contents:  |
+          [Unit]
+          Description=Cinder LVM losetup
+          DefaultDependencies=no
+          Conflicts=umount.target
+          Requires=lvm2-monitor.service systemd-udev-settle.service
+          Before=local-fs.target umount.target
+          After=var.mount lvm2-monitor.service systemd-udev-settle.service
+
+          [Service]
+          Environment=LOOPBACK_FILE=/var/home/core/cinder-volumes
+          Type=oneshot
+          ExecStart=bash -c "if [[ ! -e ${LOOPBACK_FILE} ]]; then /bin/truncate -s 10G ${LOOPBACK_FILE} && /sbin/vgcreate cinder-volumes `/sbin/losetup --show -f ${LOOPBACK_FILE}`; else /sbin/losetup -f ${LOOPBACK_FILE}; fi"
```

Comment:
What would it take to make the 10G size configurable?

Reply:
I don't think that's possible with kustomize, and I feel like we are getting sidetracked.  This is just a toy system (we are using a loop device, not even a real device) to show how it can be used.
I'll make it easier to see the size by adding the `10G` as an `Environment`

Reply:
Yeah, let's leave this along for now. I know that in tripleo the size is configurable because they need to bump it up to run tempest in some ci jobs, but that's because they have jobs that use the lvm backend.